Rakefile

```diff
@@ -8,7 +8,7 @@ require 'rake/testtask'
 
 desc 'Run all tests'
 Rake::TestTask.new('test_all') do |t|
-  t.test_files = Dir['*/tests/*.rb'] - Dir['workflow/tests/compare.rb']
+  t.test_files = Dir['*/tests/*.rb']
```

Comment:
This doesn't actually affect anything?

Reply:
There is no workflow/tests/compare.rb script anymore, so it wasn't excluding anything.